integration/airflow/openlineage/airflow/extractors/bigquery_extractor.py

```diff
@@ -80,6 +85,28 @@ def extract_on_complete(self, task_instance) -> Optional[TaskMetadata]:
             job_facets=job_facets
         )
 
+    def _get_client(self):
+        # Get client using Airflow hook - this way we use the same credentials as Airflow
+        if hasattr(self.operator, 'hook') and self.operator.hook:
+            hook = self.operator.hook
```

Comment:
I can see we're returning `Client()` on line 108. What do we do with this `hook`?